src/ingest/src/main/scala/osmesa/ingest/Ingest.scala

```diff
@@ -90,18 +91,27 @@ object Ingest extends CommandApp(
   header = "Ingest OSM ORC into GeoMesa instance",
   main = {
 
+    // Used for producing an extent from the provided string
+    implicit val readExtent: Argument[Extent] = new Argument[Extent] {
+
+      override def read(string: String): ValidatedNel[String, Extent] =
+        try { Validated.valid(Extent.fromString(string)) }
+        catch { case _: Exception => Validated.invalidNel(s"Invalid Extent: unable to parse $string") }
+
+      override def defaultMetavar: String = "extent"
+    }
     /* CLI option handling */
     val orcO = Opts.option[String]("orc", help = "Location of the ORC file to process")
     val changesetsO = Opts.option[String]("changesets", help = "Location of the ORC file containing changesets")
     val bucketO = Opts.option[String]("bucket", help = "S3 bucket to write VTs to")
     val prefixO = Opts.option[String]("key", help = "S3 directory (in bucket) to write to")
     val layerO = Opts.option[String]("layer", help = "Name of the output Layer")
     val maxzoomO = Opts.option[Int]("zoom", help = "Maximum zoom level for ingest (default=14)").withDefault(14)
-    val cacheDirO = Opts.option[String]("cache", help = "Location to cache ORC files").withDefault("")
     val pyramidF = Opts.flag("pyramid", help = "Pyramid this layer").orFalse
-    val filter = Opts.flag("filter", help = "GeoJSON multipolygon which contains all points for geoms to be processed")
+    val cacheDirO = Opts.option[String]("cache", help = "Location to cache ORC files").withDefault("")
+    val extentO = Opts.option[Extent]("extent", help = "Geographic extent for this layer").orNone
```

Comment:
What's the format of this option?

Reply:
Good call. Will add that to the documentation. This is the `fromString` I'm following: https://github.com/locationtech/geotrellis/blob/master/vector/src/main/scala/geotrellis/vector/Extent.scala#L34